packages/jest/src/migrations/update-8-10-0/update-8-10-0.ts

```diff
@@ -0,0 +1,129 @@
+import {
+  chain,
+  Rule,
+  SchematicContext,
+  Tree
+} from '@angular-devkit/schematics';
+import {
+  formatFiles,
+  insert,
+  readWorkspace,
+  updatePackagesInPackageJson
+} from '@nrwl/workspace';
+import * as ts from 'typescript';
+import * as path from 'path';
+import {
+  Change,
+  getSourceNodes,
+  InsertChange,
+  readJsonInTree,
+  ReplaceChange
+} from '@nrwl/workspace/src/utils/ast-utils';
+import { stripIndents } from '@angular-devkit/core/src/utils/literals';
+
+export default function update(): Rule {
+  return chain([
+    displayInformation,
+    updatePackagesInPackageJson(
+      path.join(__dirname, '../../../', 'migrations.json'),
+      '8.10.0'
+    ),
+    updateJestConfigs,
+    formatFiles()
+  ]);
+}
+
+function displayInformation(host: Tree, context: SchematicContext) {
+  const config = readJsonInTree(host, 'package.json');
+  if (config.devDependencies && config.devDependencies['jest-preset-angular']) {
+    context.logger.info(stripIndents`
+    \`jest-preset-angular\` 8.0.0 has restructured folders, introducing breaking changes to 
+    jest.config.js files.
+    
+    We are updating snapshotSerializers in each Angular project to include appropriate paths.
+    
+    See: https://github.com/thymikee/jest-preset-angular/releases/tag/v8.0.0
+  `);
+  }
+}
+
+function updateJestConfigs(host: Tree) {
+  const config = readJsonInTree(host, 'package.json');
+
+  if (config.devDependencies && config.devDependencies['jest-preset-angular']) {
+    const workspaceConfig = readWorkspace(host);
+    const jestConfigsToUpdate = [];
+
+    Object.keys(workspaceConfig.projects).forEach(name => {
+      const project = workspaceConfig.projects[name];
+      if (
+        project.architect &&
+        project.architect.test &&
+        project.architect.test.builder === '@nrwl/jest:jest' &&
+        project.architect.test.options &&
+        project.architect.test.options.jestConfig ===
+          project.root + 'jest.config.js'
+      ) {
+        jestConfigsToUpdate.push(project.root + 'jest.config.js');
+      }
+    });
+
+    jestConfigsToUpdate.forEach(configPath => {
+      if (host.exists(configPath)) {
+        const contents = host.read(configPath).toString();
+        const sourceFile = ts.createSourceFile(
+          configPath,
+          contents,
+          ts.ScriptTarget.Latest
+        );
+
+        const changes: Change[] = [];
+
+        getSourceNodes(sourceFile).forEach(node => {
+          if (node && ts.isStringLiteral(node)) {
```

Comment:
Awesome work here!

Unfortunately, this didn't work quite as expected for me. This is the resulting file:

```
module.exports = {
  name: 'products',
  preset: '../../jest.config.js',
  coverageDirectory: '../../coverage/apps/products',
  snapshotSerializers: [
    jest-preset-angular/build/AngularNoNgAttributesSnapshotSerializer.js'jest-preset-angular/build/AngularSnapshotSerializer.js',
    'jest-preset-angular/build/HTMLCommentSerializer.js'
  ]
};

```

Reply:
Hi @FrozenPandaz. Thanks for the review and the bug catch! 
I will update the PR asap. As for the commit authors, I can add @JoshMentzer as a co-author. I just need to find out his github email address :)

Reply:
I updated my PR to resolve this issue.